### PR TITLE
Create Calendar

### PR DESCRIPTION
**Pull request by Phil9219**
Use react-datepicker

### CODE REVIEW

File: client/src/components/Calendar/calendar.stories.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  title: "Example/Calendar",
+  component: Calendar,
+};
+
+export const CalendarDropDown = (args) => <Calendar {...args} />;
+
+// export const Login = Template.bind({});
+// Login.args = {
+//   title: "Login",
+// };
```
Comment (kschmitz123) [inline client/src/components/Calendar/calendar.stories.js]:
```suggestion

```

File: client/src/components/Calendar/calendar.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import React, { useState } from "react";
+import DatePicker from "react-datepicker";
+import "react-datepicker/dist/react-datepicker.css";
+
+export default function Calendar() {
+  const [startDate, setStartDate] = useState(new Date());
+  return (
+    <DatePicker
```
Comment (kschmitz123) [inline client/src/components/Calendar/calendar.js:8]: is it working now? 😱

File: client/src/App.js [hunk 3]
```diff
@@ -2,6 +2,8 @@ import SplashPage from "./components/Page/SplashPage";
 import AppHeader from "./components/AppHeader/AppHeader";
 import GlobalStyle from "./GlobalStyle";
 import { Buttons } from "./components/Button/Button";
+import React, { useState } from "react";
```
Comment (kschmitz123) [inline client/src/App.js]: do you need useState here?

**Comment by kschmitz123** [comment]
cool 🚀

File: client/src/components/Calendar/calendar.js (hunk 2 above)
Comment (Phil9219) [reply client/src/components/Calendar/calendar.js:8]: Yes :)

File: client/src/App.js (hunk 3 above)
Comment (Phil9219) [reply client/src/App.js]: No. Not necessary. I will delete. Thanks :)

**Pull request merged**